app/decorators/application_plan_decorator.rb

```diff
@@ -27,19 +27,10 @@ def index_table_data
       id: id,
       name: name,
       editPath: h.edit_polymorphic_path([:admin, object]),
-      applications: contracts_count,
-      applicationsPath: h.admin_service_applications_path(service, search: { plan_id: id }),
+      contracts: contracts_count,
+      contractsPath: h.admin_service_applications_path(service, search: { plan_id: id }),
```

Comment:
I'm surprised no tests broke with this change, consider adding a new test in `application_plan_decorator_test.rb` that captures this?